src/auth_api/endpoints/test.py

```diff
@@ -21,22 +25,41 @@ def __init__(self, fmt_dict: dict = None,
 
     def usesTime(self) -> bool:
         """
-        Overwritten to look for the attribute in the format dict values instead of the fmt string.
+        Look for the attribute in the format dict instead of the fmt string.
+
+        This overwrites the original function.
+
+        :return: True for when time is used. False otherwise.
+        :rtype: bool
         """
         return "asctime" in self.fmt_dict.values()
 
     def formatMessage(self, record) -> dict:
         """
-        Overwritten to return a dictionary of the relevant LogRecord attributes instead of a string.
-        KeyError is raised if an unknown attribute is provided in the fmt_dict.
+        Return a dict of the relevant LogRecord attribute instead of a string.
+
+        _extended_summary_
+
+        :param record: ??
```

Comment:
Instead of ?? use TODO as we need to identify the record and parse in a explanation